Pod/Classes/NJHNavigationBarSelectorPageViewController.m

```diff
@@ -58,6 +59,7 @@ - (void)viewWillAppear:(BOOL)animated {
         for (UIScrollView *view in self.view.subviews) {
             if ([view isKindOfClass:[UIScrollView class]]) {
                 self.pageScrollView = view;
+                self.pageScrollView.backgroundColor = self.scrollViewBackgroundColor;
```

Comment:
No you're absolutely right, what I did was an unnecessary complication. Setting the background color of the page controller's view works just as well.